junit-platform-launcher/src/module/org.junit.platform.launcher/module-info.java

```diff
@@ -30,4 +31,7 @@
 
 	uses org.junit.platform.engine.TestEngine;
 	uses org.junit.platform.launcher.TestExecutionListener;
+
+	provides org.junit.platform.launcher.TestExecutionListener
+			with org.junit.platform.launcher.listeners.FlightRecordingListener;
```

Comment:
This should be declared in `META-INF/services` so it works on the classpath as well, shouldn't it?

Reply:
Just wanted to start small: [ModularUserGuideTests](https://github.com/junit-team/junit5/blob/c7dc07b655eed11f532e983f6c178b1b02bb457b/platform-tooling-support-tests/src/test/java/platform/tooling/support/tests/ModularUserGuideTests.java#L110)

Still, I think, we should register this listener by default, i.e. programmatically. Hard-coded into the constructor of: `TestExecutionListenerRegistry`

Reply:
It should be replaced adding this listener programmatically by default.

This is just temporary hook to get this spike emitting some data.